.chloggen/remove_misleading_jaegerremotesampling_endpoint.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
As per @codeboten's comment, this should be "breaking change".

Reply:
```suggestion
change_type: breaking
```